src/identity/components/internal/WithAvatarBadge.tsx

```diff
@@ -0,0 +1,58 @@
+import type { Address } from 'viem';
+import { useVerified } from '../../hooks/useVerified';
+import { Badge } from './Badge';
+
+type DisplayBadgeProps = {
+  children: React.ReactNode;
+  address: Address;
+};
+
+function DisplayBadge({ children, address }: DisplayBadgeProps) {
```

Comment:
Let's have two files so it's easier to test and mantain

Reply:
Let me rename the component here. These two should live together.